examples/fetch-with-credentials/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "basic",
```

```suggestion
  "name": "fetch-with-credentials",
```

It causes a warning in CI.